### DISCUSSION TITLE
Live TV -  other filter options

### DISCUSSION THREAD

**Discussion by deku-m** [jellyfin/jellyfin#6417] [General]
Hi,

Im using jellyfin to also provide live tv towards my clients. 
Now im not sure if this is configured under the hood of jellyfin or in the different appz (web, android etc). But i wonder if it is possible to provide different filter options instead of only like, dislike and favorite. 

![afbeelding](https://user-images.githubusercontent.com/37069737/129197731-a126617f-8fb6-4b24-83de-b256ca05ddaf.png)

Can this be grabbed from somewhere or add your own filter list into jellyfin or maybe provided by Xteve plugin for example (Filtering there as you provide in there custom filters like NL / DE / UK etc.)

First trying it on the discussion board as im not sure if this needs to be shifted to issues.

Thanks.

**Comment by UltimateMedia** [comment]
HI Dutchie, 

do you mean how to set live tv up or use an EPG?

Greetings

**Comment by deku-m** [comment]
Its more like using different filters then the defaults. Like favorites or like or not.
Based on group for example NL or so.

As i didnt find a way to create my own filter in the gui or i missed it.
Now you can see all the channels provided un channels and can only filter on the 3 options mentioned in the screenshot.

**Comment by UltimateMedia** [comment]
I get it now. Would be nice though. Other question OT do you know a working
epg for Dutch TV?

Op zo 19 dec. 2021 15:22 schreef deku-m ***@***.***>:
